punc/include/punc/diagnostics.h

```diff
@@ -153,6 +153,7 @@ class History
 private:
     std::size_t m;
     double v, m_old, m_new, s_old, s_new;
+    double statistics[4];
```

Comment:
This is not an important feedback, but this variable could be more local.
```suggestion
```

Reply:
Ok. I forgot to remove 
` std::size_t m;
    double v, m_old, m_new, s_old, s_new;`

They should not be there.

Reply:
Right! I think if you push changes to the `statistics` branch it should get updated here as well. There also seems to be some new feature where you can accept or reject my suggestions, but I've never tested it before.

Reply:
All this is new to me, and this is probably my second or third pull request. But I'll just push the changes to `statistics`, and see how it goes.